cg/apps/orderform/excel_orderform_parser.py

```diff
@@ -152,7 +152,14 @@ def get_project_type(self, document_title: str) -> str:
         if Orderform.RML in document_title:
             return str(OrderType.RML) if analysis == self.NO_ANALYSIS else analysis
         if Orderform.MIP_DNA in document_title:
-            return str(OrderType.FASTQ) if analysis == self.NO_ANALYSIS else analysis
+            if analysis == self.NO_ANALYSIS:
+                return (
+                    str(OrderType.METAGENOME)
```

Comment:
Do we need all the str coercion now when python is 3.11 and the enum in StrEnum?